app/src/pages/explorer/sample/sample.html

```diff
@@ -3,59 +3,65 @@
     <spinner :loading="loading" :text="'Loading sample page'" />
   </div>
   <div v-else>
-    <div v-if="header" data-test="header">
-      <h1 class="visualize_header-h1 article_title u_centralize_text">Sample ID: {{ this.$route.params.label.toUpperCase() }}</h1>
-      <div class="u_centralize_text utility-color utility-line-height-sm">
-        <span class="utility-navfonticon">{{ header.sample_label.toUpperCase() }}</span>
-        <span class="utility-navfonticon">{{ header.process_label.toUpperCase() }}</span>
-        <div>DOI: {{ header.DOI }}</div>
-      </div>
-    </div>
-    <div style="max-width: 99%;">
+    <div style="max-width: 99%">
       <div class="md-layout md-gutter" :class="`md-alignment-top-center`">
         <div class="md-layout-item md-size-50">
+          <div v-if="header" data-test="header">
+            <h1 class="md-headline article_title u_centralize_text">
+              {{`${header.sample_label.toUpperCase()} ${header.process_label.toUpperCase()}`}}
+            </h1>
+          </div>
+          <template v-if="header">

```

Comment:
am curious why you went for a template here instead of just a regular div

Reply:
> am curious why you went for a template here instead of just a regular div

I thought it would be cool 🚀